deployment/bin/glassfish_administer_service.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python
```

Comment:
Can we not duplicate this file in two places? Maybe copy from the original location to here on a build?